backend/serializers/hacknight_serializer.py

```diff
@@ -0,0 +1,13 @@
+from marshmallow import Schema, fields
+
+
+class HacknightSchema(Schema):
+    class Meta:
+        fields = ('id', 'date', 'participants')
+        dump_only = ('id',)
+
+    participants = fields.Nested(
+        'ParticipantSchema',
+        exclude=('name', 'lastname', 'email', 'hacknights', 'phone'),
```

Comment:
I thought that we want to exclude only hacknigths here (just to not create circular dependency).

Reply:
I believe that You suggested me, that we shouldn't return whole participant object in hacknight schema. That why I used only with 'id' and 'github' arguments, which I changed then after Your suggestion to exclude.